packages/typespec-vscode/src/vscode-variable-resolver.ts

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
refactor the class into a separated file. From curious, why do we create a class for it? It feels to me that a utility method should be enough to do the work.